src/main/scala/com/timeout/kamon/cloudwatch/MetricsShipper.scala

```diff
@@ -0,0 +1,54 @@
+package com.timeout.kamon.cloudwatch
+
+import java.util.concurrent.{ExecutorService, Executors}
+
+import akka.pattern.pipe
+import akka.actor.{Actor, ActorLogging, Props}
+import akka.event.LoggingReceive
+import com.amazonaws.client.builder.ExecutorFactory
+import com.amazonaws.services.cloudwatch.model.{MetricDatum, PutMetricDataRequest, PutMetricDataResult}
+import com.amazonaws.services.cloudwatch.{AmazonCloudWatchAsync, AmazonCloudWatchAsyncClientBuilder}
+import com.timeout.kamon.cloudwatch.KamonSettings.{nameSpace, region}
+import com.timeout.kamon.cloudwatch.FutureUtils._
+
+import scala.collection.JavaConverters._
+import scala.concurrent.ExecutionContext
+
+
+/**
+  * Ship-and-forget. Let the java future to process the actual shipment.
+  */
+class MetricsShipper(implicit ec: ExecutionContext) extends Actor with ActorLogging {
+
+  // async aws client uses a thread pool that reuses a fixed number of threads
+  // operating off a shared unbounded queue.
+  val client: AmazonCloudWatchAsync = AmazonCloudWatchAsyncClientBuilder
+    .standard()
+    .withRegion(region)
+    .withExecutorFactory(
+      new ExecutorFactory {
+        // don't use the default thread pool which configures 50 number of threads
+        override def newExecutor(): ExecutorService = Executors.newFixedThreadPool(2)
+      }
+    ).build()
+
+
+  override def receive: Receive = LoggingReceive {
+    case ShipMetrics(metrics) =>
+      val f = client.putMetricDataAsync(
+        new PutMetricDataRequest().withNamespace(nameSpace).withMetricData(metrics.asJava)
+      )
+      f.asScala.pipeTo(self)
```

Comment:
I think maybe we'd be interested in at least logging that it failed, if we wonder where all the metrics have gone for example